physionet-django/project/templates/project/data_access_request_status_detail.html

```diff
@@ -8,5 +8,8 @@ <h1>Data Access Request - {{ project }}</h1>
         <h2>{{ access_request.data_use_title }}</h2>
         <small class="text-muted">submitted on {{ access_request.request_datetime }}{% if access_request.is_accepted %}, valid {% if access_request.valid_until %}until {{ access_request.valid_until }}{% else %}forever {% endif %}{% endif %}{% include 'project/data_access_request_badge.html' with data_access_request=access_request %}</small>
         <p>{{ access_request.data_use_purpose|safe }}</p>
+        {% if access_request.lay_summary %}
```

Comment:
even if data is not provided I think it is better to display the field to the use